packages/launchpad/src/setup/InstallDependencies.vue

```diff
@@ -1,13 +1,26 @@
 <template>
   <WizardLayout
-    :next="canNavigateForward ? t('setupPage.step.continue') : t('setupPage.install.waitForInstall')"
-    :can-navigate-forward="canNavigateForward"
+    :next="allDependenciesInstalled ? t('setupPage.step.continue') : t('setupPage.install.waitForInstall')"
+    :can-navigate-forward="allDependenciesInstalled"
     :back-fn="props.backFn"
     :next-fn="confirmInstalled"
     class="max-w-640px relative"
-    :main-button-variant="canNavigateForward ? 'primary' : 'pending'"
-    :skip-fn="!canNavigateForward ? confirmInstalled : undefined"
+    :main-button-variant="allDependenciesInstalled ? 'primary' : 'pending'"
+    :skip-fn="!allDependenciesInstalled ? confirmInstalled : undefined"
   >
+    <template
+      v-if="allDependenciesInstalled && showSuccessAlert"
+      #accessory
+    >
+      <Alert
+        v-model="showSuccessAlert"
+        class="w-full"
+        :icon="CircleCheck"
+        :title="t('setupPage.install.installationAlertSuccess')"
+        status="success"
+        dismissible
```

Comment:
What's the purpose of making this dismissable?

Reply:
If the user dismisses it, we're back in the UI state that the new banner intended to mitigate. Also, if it's not dismissable, I think we can get rid of the `showSuccessAlert` variable

Reply:
The long and short is that it's shown as dismissible within Figma. I think the updates to this view's text mitigate the weirdness better than the banner does, so I'm personally not too concerned about it being dismissible. 

We can circle back around with design in a future ticket if there's a strong preference?